content/blog/2020/01/2020-01-27-jenkins-world-lisbon-with-love-from-india.adoc

```diff
@@ -0,0 +1,111 @@
+themed party, sponsored by Sonatype. It was probably the most fun event in the entire conference. The expo hall had an entirely different look with the party lights on, 
+people wearing capes, fun events going around. There were artists dressed in a Bumblebee, a Batman, a Superman, a Supergirl, a Thor and more superhero 
+costumes! I was previously made aware of the interesting parties at Jenkins World but the experience was very different. People from all over the world had 
+come together to celebrate the 15 years of success of an open source software. After partying from 5 to 7 we went back to the hotel. I spent some time 
+to prepare the slides for the next day presentation and went to bed.
+
+image::/images/post-images/2020/01-gsoc-jenkins-world-parichay/superhero-party.jpg[A Bumblebee at Super Hero Party]
+
+== Day 3
+
+image::/images/post-images/2020/01-gsoc-jenkins-world-parichay/fun-run.jpg[6K Jenkins World Fun Run]
+
+The third and final day of conference began with Jenkins World Fun Run. I missed the keynote for being late and other setup required for the 
+presentation. My laptop was broken so had to do all the setup for demo on a friend’s laptop. The situation felt like a Jenkins admin under fire 
+for a production bug. After being under pressure for a while, took a break to admire the developer comics and had a chat with the graffiti painter. 
+During lunch it was time for the GSoC presentation at the Cloudbees community booth. All our presentation went well and we also interacted with real users. 
+Then we had the GSoC Team pic at the Cloudbees community booth. Later Abhyudaya and me gave our presentation at the lightning talks as well upon Mark Waite’s 
+request. The event concluded with emotional goodbyes. 
+
+image::/images/post-images/2020/01-gsoc-jenkins-world-parichay/dev-comics.jpg[Developer Comics]
+
+image::/images/post-images/2020/01-gsoc-jenkins-world-parichay/graffiti-painter.jpg[Graffiti Wall]
+
+All GSoC Students were invited for dinner at Corinthia Lisboa's Soul Garden restaurant. The party comprised of Oleg, Mark and his lovely wife, Liam, Tracy, 
+Alyssa and Olivier. We had a very nice conversation and I had a very delicious Bacalhau (cod fish) dish. Then bid final goodbye to everybody.
+
+image::/images/post-images/2020/01-gsoc-jenkins-world-parichay/project-presentation.jpg[Presenting my GSoC project work]
+
+image::/images/post-images/2020/01-gsoc-jenkins-world-parichay/gsoc-team.JPG[GSoC Team Picture]
+
+It was a wonderful experience in a wonderful country among wonderful people. Hats off to the management team lead by Alyssa Tong and co. An event this big was
+carried out without any hiccups! Everybody contributed their part to the event which made it very interactive and fun. Checkout some of my swags:
+
+image::/images/post-images/2020/01-gsoc-jenkins-world-parichay/swags-dwjw.png[Swag from Jenkins World Lisbon]
+
+Thank you Jenkins and Google Summer of Code :)
```

Comment:
Would be also great to say thanks to CloudBees for the conference ticket sponsorship